src/main/webapp/oscarEncounter/oscarConsultationRequest/DisplayDemographicConsultationRequests.jsp

```diff
@@ -75,76 +83,119 @@ theRequests.estConsultationVecByDemographic(LoggedInInfo.getLoggedInInfoFromSess
 
 <html:html lang="en">
 <head>
-<script type="text/javascript" src="<%= request.getContextPath() %>/js/global.js"></script>
 <title><bean:message
 	key="oscarEncounter.oscarConsultationRequest.DisplayDemographicConsultationRequests.title" />
 </title>
 <html:base />
 
 <!--META HTTP-EQUIV="Refresh" CONTENT="20;"-->
 
-<link rel="stylesheet" type="text/css" media="all" href="../share/css/extractedFromPages.css"  />
+<!-- jquery -->
+	<script src="${pageContext.request.contextPath}/library/jquery/jquery-3.6.4.min.js"></script>
+	<script src="${pageContext.request.contextPath}/library/bootstrap/3.0.0/js/bootstrap.min.js"></script>
+	<script src="${pageContext.request.contextPath}/library/DataTables/datatables.min.js"></script><!-- 1.13.4 -->
 
+<!-- css -->
+	<link href="${pageContext.request.contextPath}/library/bootstrap/3.0.0/css/bootstrap.min.css" rel="stylesheet"><!-- Bootstrap 2.3.1 -->
+	<link href="${pageContext.request.contextPath}/library/bootstrap/3.0.0/assets/css/DT_bootstrap.css" rel="stylesheet">
+	<link href="${pageContext.request.contextPath}/library/DataTables-1.10.12/media/css/jquery.dataTables.min.css" rel="stylesheet">
 
+	<style>
+		.MainTable .dataTables_length label {
+			display: flex;
+			align-items: center;
+			gap: 5px;
+		}
+		.MainTable .dataTables_length select {
+			display: inline-block;
+			width: auto;
+		}
 
+		.MainTable .dataTables_filter label {
+			display: flex;
+			align-items: center;
+			gap: 5px;
+		}
+		.MainTable .dataTables_filter input {
+			display: inline-block;
+			width: auto;
+		}
 
+		.MainTable tbody > tr > td {
+			border-top: none;
+		}
 
-</head>
-<script language="javascript">
-function BackToOscar()
-{
-       window.close();
-}
-function popupOscarRx(vheight,vwidth,varpage) { //open a new popup window
-  var page = varpage;
-  windowprops = "height="+vheight+",width="+vwidth+",location=no,scrollbars=yes,menubars=no,toolbars=no,resizable=yes,screenX=0,screenY=0,top=0,left=0";
-  var popup=window.open(varpage, "<bean:message key="oscarEncounter.oscarConsultationRequest.DisplayDemographicConsultationRequests.msgConsReq"/>", windowprops);
-  //if (popup != null) {
-  //  if (popup.opener == null) {
-  //    popup.opener = self;
-  //  }
-  //}
-}
-function popupOscarConS(vheight,vwidth,varpage) { //open a new popup window
-  var page = varpage;
-  windowprops = "height="+vheight+",width="+vwidth+",location=no,scrollbars=yes,menubars=no,toolbars=no,resizable=yes,screenX=0,screenY=0,top=0,left=0";
-  var popup=window.open(varpage, "<bean:message key="oscarEncounter.oscarConsultationRequest.ConsultChoice.oscarConS"/>", windowprops);
-  window.close();
-}
+		tr.MainTableTopRow  td {
+			padding: 0 8px 0 8px !important;
+		}
+	</style>
+
+<script>
+	jQuery(document).ready( function () {
+	    jQuery('#consultTable').DataTable({
+			"lengthMenu": [ [25, 50, 100, -1], [25, 50, 100, "<bean:message key="oscarEncounter.LeftNavBar.AllLabs"/>"] ],
+			"order": [[6,'desc']],
```

Comment:
This should be changed to 7 when #64 is merged since it adds a column (consultant)